app/src/main/java/org/mozilla/focus/utils/DialogUtils.java

```diff
@@ -105,6 +112,10 @@ public void onCancel(DialogInterface dialogInterface) {
         });
 
         View dialogView = LayoutInflater.from(context).inflate(R.layout.layout_share_app_dialog, (ViewGroup) null);
+
+        final TextView textView = dialogView.findViewById(R.id.share_app_dialog_textview_title);
+        textView.setText(context.getString(R.string.share_app_dialog_text_title, context.getString(R.string.app_name)));
```

Comment:
NIT: remove the text in layout file so devs won't attempt to change these text there?